### DISCUSSION TITLE
Spell checking Swedish in 3.5.0

### DISCUSSION THREAD

**Discussion by MarcLouvet** [SafeExamBrowser/seb-win-refactoring#656] [Q&A]
Hi! Is there anyway one can enable spell checking for a different language than english? In 3.5.0 the option for other languages is removed. Has the feature moved elsewhere in the config tool? If not, (when) can we expect spell checking in swedish to be enabled again?

Best, Marcus

**Comment by dbuechel** [comment]
At the moment, we only support the built-in functionality of Chromium. If there is an easy way to enable other languages, then we could look into that as a feature request for an upcoming version, otherwise we're most unfortunately extremely busy with other features and functionalities and most likely won't have the time to implement a similar option as we had it previously.
